components/application-deployers/org.wso2.carbon.application.deployer.synapse/src/main/java/org/wso2/carbon/application/deployer/synapse/SynapseAppDeployer.java

```diff
@@ -307,11 +304,11 @@ private void deploySynapseLibrary(List<Artifact.Dependency> artifacts,
                                         String libraryPackage = artifactName.substring(1, artifactName.lastIndexOf("}"));
                                         updateStatus(artifactName, libName, libraryPackage, ServiceBusConstants.ENABLED, axisConfig);
                                     }
+                                } else {
+                                    log.error("Error while getting qualified name of the synapse library.");
```

Comment:
Put exact cause of the error like "artifact name is null" with the log message. It will be beneficial in troubleshooting an issue.